coordinator/src/node/event_post_processing.rs

```diff
@@ -0,0 +1,47 @@
+use crate::db;
+use crate::node::Node;
+use crate::routing_fee::models::NewRoutingFee;
+use lightning::util::events::Event;
+use tokio::sync::watch;
+
+pub fn spawn(node: Node, mut node_event_receiver: watch::Receiver<Option<Event>>) {
+    tokio::spawn(async move {
```

Comment:
nit: i would preferred function not to spawn, but leave that to the calling function.